tests/test_transforms.py

```diff
@@ -264,9 +263,10 @@ def test_object_construct() -> None:
 
 def test_parse_json() -> None:
     assert (
-        sqlglot.parse_one("""insert into table1 (name) select parse_json('{"first":"foo", "last":"bar"}')""")
-        .transform(parse_json)
-        .sql(dialect="duckdb")
+        sqlglot.parse_one(
+            """insert into table1 (name) select parse_json('{"first":"foo", "last":"bar"}')""",
+            read="snowflake",
+        ).sql(dialect="duckdb")
         == """INSERT INTO table1 (name) SELECT JSON('{"first":"foo", "last":"bar"}')"""
     )
 
```

Comment:
may as well delete the whole test!
```suggestion
```

Reply:
deleted @https://github.com/tekumara/fakesnow/pull/62/commits/9929d0c3638851829acfe741c5bfea97733571ec